recipes/apache-airflow-providers-postgres/0001-rename-psycopg2-binary.patch

```diff
@@ -0,0 +1,22 @@
+diff -ruN apache-airflow-providers-postgres-1.0.1/apache_airflow_providers_postgres.egg-info/requires.txt apache-airflow-providers-postgres-1.0.1-patch/apache_airflow_providers_postgres.egg-info/requires.txt
+--- apache_airflow_providers_postgres.egg-info/requires.txt	2021-02-08 21:37:18.000000000 +0100
++++ apache_airflow_providers_postgres.egg-info/requires.txt	2021-02-21 23:09:09.388749931 +0100
+@@ -1,5 +1,5 @@
+ apache-airflow>=2.0.0
+-psycopg2-binary>=2.7.4
++psycopg2>=2.7.4
+
+ [amazon]
+ apache-airflow-providers-amazon
+diff -ruN apache-airflow-providers-postgres-1.0.1/setup.py apache-airflow-providers-postgres-1.0.1-patch/setup.py
+--- setup.py	2021-02-08 21:37:14.000000000 +0100
++++ setup.py	2021-02-21 23:17:28.853438102 +0100
+@@ -57,7 +57,7 @@
+         ),
+         zip_safe=False,
+         include_package_data=True,
+-        install_requires=['apache-airflow>=2.0.0', 'psycopg2-binary>=2.7.4'],
++        install_requires=['apache-airflow>=2.0.0', 'psycopg2>=2.7.4'],
```

Comment:
We don't provide `psycopg2-binary` on conda-forge.  It appears to just be an unoptimized, simplified version of `psycopg2`.  This patch is to make `pip check` happy.

Reply:
`psycopg2-binary` should be the same as `psycopg2`, the only difference is that the former provides wheels and the other one doesn't and I would expected that it provides the same metadata. If there is a difference in Python package metadata between both, we should add that to `psycopg2` package.

Reply:
Yes, without this change, `pip check` fails so it sounds like maybe `psycopg2-binary` should be created in some format that `pip check` will recognize (i.e. not just a `conda` metapackage).

Reply:
my dream is something that _doesn't_ require pip to do what `pip check` does... raise your hand if you've missed a `setuptools` dependency because of `pkg_utils` :hand: 

for now, because `conda`, i think this is a way to proceed, and is _unlikely_ to break anything, so long as the patch breaks when/if this number changes.